src/views/settings/Auth/const.js

```diff
@@ -22,136 +23,108 @@ export function getOrgSelect2Meta() {
-      title: vm.$t('WeCom'),
+      title: i18n.t('WeCom'),
       authKey: 'AUTH_WECOM',
-      description: '',
-      enabled: false
+      description: ''
     },
     {
+      name: 'DingTalk',
       logo: require('@/assets/img/auth/dingtalk_logo.webp'),
-      title: vm.$t('DingTalk'),
+      title: i18n.t('DingTalk'),
       authKey: 'AUTH_DINGTALK',
-      description: '',
-      enabled: false
+      description: ''
     },
     {
+      name: 'FeiShu',
       logo: require('@/assets/img/auth/feishu_logo.png'),
-      title: vm.$t('FeiShu'),
+      title: i18n.t('FeiShu'),
       authKey: 'AUTH_FEISHU',
-      description: '',
-      enabled: false
+      description: ''
     },
     {
+      name: 'Lark',
       logo: require('@/assets/img/auth/lark_logo.png'),
-      title: vm.$t('Lark'),
+      title: i18n.t('Lark'),
       authKey: 'AUTH_LARK',
-      description: '',
-      enabled: false
+      description: ''
     },
     {
+      name: 'Slack',
       logo: require('@/assets/img/auth/slack_logo.png'),
-      title: vm.$t('Slack'),
+      title: i18n.t('Slack'),
       authKey: 'AUTH_SLACK',
-      description: '',
-      enabled: false
+      description: ''
     },
     {
+      name: 'Radius',
       logo: require('@/assets/img/auth/radius_logo.png'),
-      title: vm.$t('Radius'),
+      title: i18n.t('Radius'),
       authKey: 'AUTH_RADIUS',
-      description: '',
-      enabled: false
+      description: ''
     }
   ]
 
   const authItems = [
     {
+      name: 'LDAP',
       logo: require('@/assets/img/auth/ldap_logo.png'),
-      title: vm.$t('Ldap'),
+      title: i18n.t('Ldap'),
       authKey: 'AUTH_LDAP',
-      description: '',
-      enabled: true
+      description: ''
     },
     {
+      name: 'CAS',
       logo: require('@/assets/img/auth/cas_logo.png'),
-      title: vm.$t('CAS'),
+      title: i18n.t('CAS'),
       authKey: 'AUTH_CAS',
-      description: '',
-      enabled: true
+      description: ''
     },
     {
+      name: 'Passkey',
       logo: require('@/assets/img/auth/passkey_logo.png'),
-      title: vm.$t('Passkey'),
+      title: i18n.t('Passkey'),
       authKey: 'AUTH_PASSKEY',
-      description: '',
-      enabled: true
+      description: ''
     }
   ]
 
   if (store.getters.hasValidLicense) {
     // 把 AUTH_LDAP_HA 加到 authItems 中的 AUTH_LDAP 后面
-    const ldapIndex = authItems.findIndex(item => item.authKey === 'AUTH_LDAP')
-
-    if (ldapIndex !== -1) {
-      authItems.splice(ldapIndex + 1, 0, ...ldapHABackends)
-    }
-
-    authItems.push(...extraBackends)
+    authItems.push(...xpackBackends)
   }
-
-  authLocalStorage.set('authItems', authItems)
-
   return authItems
 }
 
-export const setAuthItem = (value, authKey) => {
-  const authItems = authLocalStorage.get('authItems')
-  const authItem = authItems.find(item => item.authKey === authKey)
-
-  if (authItem) {
-    authItem.enabled = value
-    authLocalStorage.set('authItems', authItems)
-  }
-}
-
```

Comment:
There are some inconsistencies in the code:

  In `export async componentDidMount()` you need to specify an import statement to include `<ErrorBoundary>`.

  You should move `"const xpackBackends = ["` outside this block since `return [x]`.

In order to fix them, modify the lines accordingly. Also, please note that there is no proper documentation provided for these functions; they might have been intended for a different purpose or may not be available at all based on the context of your application.